File: 217-ContainsDuplicate/217-ContainsDuplicate.py
# Last updated: 9/27/2025, 11:23:51 AM
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        dct={}
        for i in nums:
            if i not in dct.keys():
                dct[i]=1
            else:
                return True
        return False

File: 217-ContainsDuplicate/217-ContainsDuplicate.cpp
// Last updated: 4/4/2026, 12:42:11 AM
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        bool fnd=false;
        int sz=nums.size()-1;
        sort(nums.begin(),nums.end());
        for (int i=0;i<sz;i++){
            if (nums[i]==nums[i+1]){
                fnd=true;
                break;
            }
        }
        return fnd;
    }
};